stable/risk-advisor/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+1. Check how the cluster state would change after adding provided pods by running these command:
+	curl -XPOST SERVICE_URL:11111/advise -H "Content-type: application/json" -d @PATH_TO_JSON_WITH_PODS
```

Comment:
This is very little information. URL/port should not be hard-coded.

Here's a good example: https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/templates/NOTES.txt